providers.json

```diff
@@ -2,6 +2,9 @@
     {
         "name": "Timothy Garrick",
         "city": "Boston",
-        "state": "Massachusetts"
+        "state": "Massachusetts",
+        "fhir_url": "http://52.39.26.206:9000/api/fhir",
+        "oauth_url": "http://52.39.26.206:9000/api/oauth",
```

Comment:
In theory fhir_url is the only thing that should be stored here - the others can be derived from the FHIR /metadata endpoint